src/Pages/NotificationPage/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import Navbar from '../../Components/Navbar';
+// import styles from './style.module.scss';
+// import Icon from '../../Components/Icon';
+
+function NotificationPage() {
+  // const ladyWidth = 400;
+  // const bananaWidth = 60;
+
+  useEffect(() => {
+    document.body.classList.add('theBody');
```

Comment:
@timtran007 I know this was done somewhere else in the code and I have no idea why it was done like this! 😂  
could you please remove this and see if you can just hard-code the class `theBody` on the `<body>` element (or wherever it gets added) let me know if you want some help or clearer instructions on that

Reply:
Done!